types.go

```diff
@@ -190,6 +167,15 @@ const (
 	AttachModifyReturn
 	AttachLSMMac
 	AttachTraceIter
+	AttachCgroupInet4GetPeername
+	AttachCgroupInet6GetPeername
+	AttachCgroupInet4GetSockname
+	AttachCgroupInet6GetSockname
+	AttachXDPDevmap
```

Comment:
I think 'dev map' is two words. Like 'CPU map'.

Reply:
Done, thanks.